### ISSUE TITLE
Perl::Critic::Policy::ValuesAndExpressions::ProhibitLeadingZeros strange behavior with sysopen

### ISSUE THREAD

**Issue by claviola** [Perl-Critic/Perl-Critic#789]
This is okay for perlcritic:

```
my $handle;
sysopen $handle, $filename, O_RDWR | O_BINARY | O_CREAT, 0640;
```

While this isn't:

`sysopen my $handle, $filename, O_RDWR | O_BINARY | O_CREAT, 0640;`

**Comment by petdance** [comment]
Interesting.  Thanks for reporting it.

**Comment by petdance** [comment]
This has been fixed and will go in the next release after 1.140.
